### DISCUSSION TITLE
Iyzico payment. Return page error. 404

### DISCUSSION THREAD

**Discussion by cengkaplan33** [serenity-is/Serenity#6047] [General]
Hi peope , I finished Iyzico payment system to .netcore  serenity.  After payment operation bank must return my fail or success url.
After 3d payment end , bank return my url which is "http://127.0.0.1:5000/PosPaymentThreeDPost".
When Iyzico(bank) return my address it take "404 error"
But I can call my addres from another application or entering chrome browser. and it works.

Can anyone give me a solution or idea about this situation.

My code is like below.
    [Route("PosPayment/[action]")]
    public class PosPaymentController : Controller
    {
        [HttpPost,HttpGet]
        [Route("~/PosPaymentThreeDPost")]
        public ActionResult PosPaymentThreeDPost()
        {
            return View(MVC.Views.Site.PosPayment.PosPaymentPost,new PosPaymentPostModel(){});
        }
}

error :

![image](https://user-images.githubusercontent.com/3457947/137760360-3f1f58cb-c5db-4fdd-87c3-6c5c3b328b38.png)

**Comment by VictorTomaili** [comment]
you wrote the wrong URL for return. Check your URL for the `127.0.0.1:5000;/Pos` section has a semicolon, and also it doesn't relate to the serenity platform.

**Reply by cengkaplan33** [reply]
victor it is image problem . My address is http://127.0.0.1:5000/PosPaymentThreeDPost
I manipulated the picture .

**Reply by cengkaplan33** [reply]
thanks guys . I solved with IgnoreAntiforgeryToken .
